hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/fair/FSAppAttempt.java

```diff
@@ -1123,6 +1201,13 @@ public Resource getDemand() {
     return demand;
   }
 
+  /**
+   * Get the current app's unsatisfied demand.
+   */
```

Comment:
Trivial, unambiguous @Private method that is package-private. The @return is not of much use.